Directory.Build.props

```diff
@@ -1,17 +1,26 @@
 <Project>
   <Import Project="version.props" />
   <Import Project="JustSaying.Models.version.props" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
+    <PackageReference Include="OpenCover" Version="4.6.519" PrivateAssets="All" />
+    <PackageReference Include="ReportGenerator" Version="3.1.2" PrivateAssets="All" />
+  </ItemGroup>
   <PropertyGroup>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

Comment:
Is this actually required?

Reply:
It's not in this case because the pdbs are embedded